MFVNeutralino/python/Vertexer_cfi.py

```diff
@@ -46,23 +50,35 @@
 
 mfvVertices = cms.EDProducer('MFVVertexer',
                              kvr_params = kvr_params,
+                             do_track_refinement = cms.bool(False), # track refinement  
+                             resolve_split_vertices_loose = cms.bool(False), # Jordan merging 
```

Comment:
Let's give this a different / more specific name, to be easier to remember what we mean here.

Reply:
Done.